source/index.txt

```diff
@@ -22,16 +16,60 @@ The Node driver documentation is organized as follows:
   /api-documentation
   /faq
   /issues-and-help
+  /compatibility
 
-Compatibility
--------------
 
-MongoDB Compatibility
-~~~~~~~~~~~~~~~~~~~~~
+Introduction
+------------
+The official MongoDB Node.js driver allows Node.js applications to connect and perform queries with MongoDB.
+
+.. note::
+   
+   These docs are for version 3.5 of the MongoDB Node.js driver. If
+   you're looking for an older version of the MongoDB Node.js driver
+   docs, `see the GitHub pages
+   <https://mongodb.github.io/node-mongodb-native/>`_. For the main
+   MongoDB documentation, see the :manual:`MongoDB Manual </>`.
+
+Quick Start
+-----------
+Learn how to quickly get a Node.js application working with MongoDB Atlas through the `Quick Start </quick-start/>`_ section
+
+Fundamentals
+------------
+The `Fundamentals </fundamentals>`_ section defines the following core concepts involved with using the MongoDB
+Node.js driver: 
 
-.. include:: /includes/mongodb-compatibility-table-node.rst
+* CRUD: Learn how to read and write data to MongoDB using Node.js
+* Authentication: Learn how to log a user in through various community and enterprise mechanisms
+* Logs: Learn how MongoDB logs operations and how to work with them
+* Monitoring: Learn how to monitor server events
 
-Language Compatibility
-~~~~~~~~~~~~~~~~~~~~~~
+Usage Examples
+--------------
+Learn how to use common MongoDB Node.js driver methods through brief
+explanation and opinionated code snippets in the `Usage Examples </usage-examples/overview>`_ section.
+This section is best suited for new users to the driver who wants to
+implement common driver methods quickly. 
 
-.. include:: /includes/language-compatibility-table-node.rst
+API
+---
+See the `API </api-documentation/>`_  section if you are looking for the API
+reference for MongoDB Node.js driver methods. 
```

Comment:
"API reference" might not have a ton of meaning to new users or could even be confusing (e.g. "oh like a REST API"). Maybe we could give a bit more color as to what's inside. e.g. detailed and thorough information for classes, methods, and configuration objects

Reply:
Altered to: 

"See the `API </api-documentation/>`_  section if you are looking for
technical information for classes, methods, and configuration objects"